common/util.go

```diff
@@ -176,7 +176,21 @@ func ListMountPoints() ([]string, error) {
 	// TODO: Add support to list current mounts
 	// We cannot list mount points like we do on Linux
 	if runtime.GOOS == "windows" {
```

Comment:
This function seems to break pretty neatly by platform (no code is shared), so it might be nice to move this whole function out of this file, and instead put the two implementations into separate platform-specific files: util_windows.go and util_linux.go.

Reply:
The other function, GetCurrentUser(), would need to be moved for this as well. I would consider moving these to a separate windows file as a separate task for the backlog. what do you think?

Reply:
You could split `GetCurrentUser` into two separate implementations, but there are at least a few lines of code shared by the two, which makes a case for leaving it the way it is.
In the case of ListMountPoints, nothing is shared.
Moving the functions to separate files could be another PR, but I think if we're going to do it, we should just go ahead and do it in this PR. It's not a big lift.

Reply:
this was a misunderstanding of having to create linux and windows files and split up all function windows sections into the windows file. util_windows.go already exists and I'll just move the windows portion of the function to that file.